#!/usr/bin/env python

PACKAGE = "turtle_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_low",    int_t,    0, "Low hue", 10,  0, 179)
gen.add("h_up",    int_t,    0, "Up hue", 160,  0, 179)
gen.add("s_low",    int_t,    0, "Low Sat", 10,  0, 255)
gen.add("s_up",    int_t,    0, "Up Sat", 250,  0, 255)
gen.add("v_low",    int_t,    0, "Low Value", 10,  0,255)
gen.add("v_up",    int_t,    0, "Up Value", 250,  0, 255)


exit(gen.generate(PACKAGE, "turtle_controller_hsv", "HSVParams"))
